src/components/SlateEditor/plugins/embed/SlateImage.jsx

```diff
@@ -112,6 +112,7 @@ class SlateImage extends React.Component {
               <img
                 src={src}
                 alt={embed.alt}
+                style={{ minWidth: '716px' }}
```

Comment:
Skal denne være så spesifikk?

Reply:
Raskeste måten men litt shady. 716px er bredden som større bildene blir satt til, altså bredden til conteineren som inneholder bilder.

Reply:
Er container bredden den samme på alle enheter? 
Ser litt hacky ut ja, har ikke frontend-packages en eller annen verdi som er definert fra før av som du kan bruke? Basert på screen size f.eks

Reply:
Ser nå at det fungerer ikke på mobilen, ting blir skalert for mye. Skal gjøre det mer robust